### PR TITLE
add rtmpdump

### PR DESCRIPTION
**Pull request by Dentrax**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)
- [ ] If non-streamed package names no longer built, open PR to withdraw them (see [WITHDRAWING PACKAGES](https://github.com/wolfi-dev/os/blob/main/WITHDRAWING_PACKAGES.md))

#### For package updates (renames) in the base images

When updating packages part of base images (i.e. cgr.dev/chainguard/wolfi-base or ghcr.io/wolfi-dev/sdk)
- [ ] REQUIRED cgr.dev/chainguard/wolfi-base and ghcr.io/wolfi-dev/sdk images successfully build
- [ ] REQUIRED cgr.dev/chainguard/wolfi-base and ghcr.io/wolfi-dev/sdk contain no obsolete (no longer built) packages
- [ ] Upon launch, does `apk upgrade --latest` successfully upgrades packages or performs no actions

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

File: rtmpdump.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+  - name: librtmp
+    description: RTMP library
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/usr/lib
+          mv librtmp/*.so* "${{targets.subpkgdir}}"/usr/lib/
+
+update:
+  enabled: false
```
Comment (EyeCantCU) [inline rtmpdump.yaml:65]: We can use the git poller here

File: rtmpdump.yaml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  version: 2.6_git20241004
+  epoch: 0
+  description: rtmpdump is a toolkit for RTMP streams
+  copyright:
+    - license: LGPL-2.0-only
+
+environment:
+  contents:
+    packages:
+      - automake
```
Comment (EyeCantCU) [inline rtmpdump.yaml:13]: Included by the make pipeline

**Review by EyeCantCU — CHANGES_REQUESTED** [review]
Looks good. We can use the git poller instead of disabling the update check though

File: rtmpdump.yaml (hunk 1 above)
Comment (Dentrax) [reply rtmpdump.yaml]: Thanks, done!

**Review by EyeCantCU — APPROVED** [review]

**Pull request merged**